docs/usage.md

```diff
@@ -3,130 +3,441 @@
 
-![FortiAI response for a Jinja query](../docs/res/FortiAIJinja.png)
+2. **Modify Playbook Outline**:
+   - Adjust the playbook outline as needed, such as changing the integration tool from VirusTotal to IPStack.      
+
+    ![FortiAI - Suggested playbook outline](../docs/res/FortiAIPBGeneration.png)
+
+3. **Generate Playbook**:
+   - Once satisfied with the outline, click **Ok, Generate Playbook**.
+
+>[!NOTE]
+>If a step uses a connector that is not installed, FortiAI informs you with a message indicating that the specific step requires a connector that is not installed on the system.
+
+4. **Monitor Playbook Generation**:
+   - A loader image indicates the progress while FortiAI is generating playbook steps.
+   
+5. **Review Generated Playbook**:
+   - After the steps have been generated, a suggested playbook block appears in the *Playbook Designer*. You can review each step by double-clicking it and examining the details.
+
+    ![Reviewing a playbook step generated by FortiAI](../docs/res/FortiAIGeneratedPb_StepReview.png)
+
+6. **Integrate and Finalize Playbook**:
+   - Connect the reviewed AI-generated playbook blocks to the main playbook flow to create a complete workflow.
+
+The ability to generate playbooks through OpenAI's assistant represents a significant advancement in playbook design, offering immense assistance to those new to developing playbooks. By following the above steps, administrators can efficiently create and manage robust playbooks to address various threat mitigation scenarios.
+
+## Navigating Page
+
+Navigating pages within the FortiAI assistant allows users to access various modules and records efficiently. This functionality enhances workflow efficiency and helps users locate and manage the required information across the system.
+
+### Use Case Example: Accessing Records
+
+One common use case involves accessing specific records quickly. For instance, a security analyst needs to review the details of a particular alert or incident. By issuing navigation commands, the analyst can access the required records without unnecessary delays.
+
+### Navigating a Page Using FortiAI Assistant
+
+1. **Initiate Navigation**:
+   - Issue a command to navigate to a specific page or module. For example:
+
+    >*Navigate to the Alerts module*
+
+    or
+
+    >*Navigate to Content Hub*
+
+   - The assistant directs you to the requested module, such as Alerts, Incidents, Indicators, or in the above case &ndash; to Content Hub.
+
+2. **Access a Specific Record**:
+   - Provide details such as the record's identifier or name to access a particular record. For example:
+
+    >*Open the record with ID [ALERT-ID]*
+
+   - The assistant locates and displays the record for you.
+
+Adjust the navigation command to suit your needs, such as specifying a different module or record.
+
+### Review Navigation Commands
```

Comment:
What is a use of "Review Navigation Commands" section?

Reply:
For consistency